docs/docs/PB/Verbali_Interni/_category_.json

```
@@ -0,0 +1,4 @@
+{
+  "label": "PB",
+  "position": 2
+}
```

Questo file definisce come Docusaurus interpreta la cartella. In questo caso deve essere un indice:
```suggestion
{
  "label": "Verbali Interni",
  "link": {
    "type": "generated-index",
    "description": "Tutti i verbali interni prodotti durante le riunioni svolte in preparazione alla PB."
  }
}
```